### PR TITLE
literature: handle unicode in new_ticket_context

### PR DESCRIPTION
**Pull request by chris-asl**
Sentry: https://sentry.cern.ch/inspire-sentry/inspire-labs/group/820778/

Signed-off-by: Chris Aslanoglou <chris.aslanoglou@gmail.com>

### CODE REVIEW

File: tests/unit/literaturesuggest/test_literaturesuggest_tasks.py [hunk 1]
```diff
@@ -94,6 +94,53 @@ def test_new_ticket_context():
...
 
 
+def test_new_ticket_context_handles_unicode():
+    data = {
+        'titles': [
+            {
+                u'title': 'Actual title contained: é',
+            },
+        ],
+        'external_system_numbers': [
```
Comment (jacquerie) [inline tests/unit/literaturesuggest/test_literaturesuggest_tasks.py]: This key is not relevant for the current test, so it should be removed.

File: tests/unit/literaturesuggest/test_literaturesuggest_tasks.py [hunk 2]
```diff
@@ -94,6 +94,53 @@ def test_new_ticket_context():
...
+            {
+                'value': 'bar',
+            },
+        ],
+    }
+    extra_data = {
+        'formdata': {
```
Comment (jacquerie) [inline tests/unit/literaturesuggest/test_literaturesuggest_tasks.py]: This key is not relevant for the current test, so it should be removed.

File: tests/unit/literaturesuggest/test_literaturesuggest_tasks.py [hunk 3]
```diff
@@ -94,6 +94,53 @@ def test_new_ticket_context():
     assert expected == result
 
 
+def test_new_ticket_context_handles_unicode():
+    data = {
+        'titles': [
+            {
+                u'title': 'Actual title contained: é',
```
Comment (jacquerie) [inline tests/unit/literaturesuggest/test_literaturesuggest_tasks.py]: You have the actual string that caused the error on Sentry, why invent another? Just use "Chocs caractéristiques et ondes simples exceptionnelles pour les systèmes conservatifs à intégrale d'énergie: forme explicite de la solution".

File: tests/unit/literaturesuggest/test_literaturesuggest_tasks.py [hunk 4]
```diff
@@ -94,6 +94,53 @@ def test_new_ticket_context():
...
+        ],
+        'user_comment': [
+            'baz',
+        ],
+        'references': [
+            'qux',
+        ],
+        'object': obj,
+        'subject': u'Your suggestion to INSPIRE: Actual title contained: é',
+    }
```
Comment (jacquerie) [inline tests/unit/literaturesuggest/test_literaturesuggest_tasks.py]: `expected` should be amended accordingly.

File: inspirehep/modules/literaturesuggest/tasks.py [hunk 5]
```diff
@@ -218,8 +218,8 @@ def _is_arxiv_url(url):
 
 def new_ticket_context(user, obj):
     """Context for literature new tickets."""
-    title = get_title(obj.data)
-    subject = "Your suggestion to INSPIRE: {0}".format(title)
+    title = get_title(obj.data).decode('utf-8')
```
Comment (jacquerie) [inline inspirehep/modules/literaturesuggest/tasks.py]: I forgot to comment on the most important part: why is this needed?

File: tests/unit/literaturesuggest/test_literaturesuggest_tasks.py (hunk 3 above)
Comment (chris-asl) [inline tests/unit/literaturesuggest/test_literaturesuggest_tasks.py]: Took me a while to find where the data is (_didn't know we could see these_) -  changing that now.

File: inspirehep/modules/literaturesuggest/tasks.py (hunk 5 above)
Comment (chris-asl) [inline inspirehep/modules/literaturesuggest/tasks.py]: That was erroneous and it was required by this `u'title':` that I had typed in test `data`.
